packages/mdc-dialog/mdc-dialog.scss

```diff
@@ -69,6 +69,7 @@
 .mdc-dialog__container {
   display: flex;
   flex-direction: row; // IE 11
+  justify-content: space-around;
```

Comment:
Is this the magic property that fixes something in Safari?

If so, can you add an inline comment explaining why it's there? I'm not sure what issue you're seeing.